cmd/minishift/cmd/start_preflight.go

```diff
@@ -65,30 +69,36 @@ func preflightChecksBeforeStartingHost() {
 		configCmd.WarnDeprecationCheck.Name,
 		"")
 
-	// Conectivity logic
+	// Get OpenShift version
+	requestedOpenShiftVersion, err := getOpenShiftVersion()
```

Comment:
here you can ignore the err since it already handled in the start if you put that block as I suggested.